baselines/webworker.generated.d.ts

```diff
@@ -399,9 +399,9 @@ declare var Event: {
 }
 
 interface EventTarget {
-    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    addEventListener(type: string, listener?: EventListenerOrEventListenerObject<this>, useCapture?: boolean): void;
```

Comment:
in general we do not want to set `this` as it turns all the types that inherit from EventTarget (almost all DOM types) as generic, and every time you reference them we make an instantiation, more memory and more time spent checking the default lib file.

can we make EventTarget<T extends EventTarget> and pass the current type through?